week_1/week_1.txt

```diff
@@ -0,0 +1,7 @@
+2 APRIL
+    HTML Tags
```

Comment:
please provide more info which all tags you have learned

Reply:
@abhi-codal Yes will update this in next commit.